components/content/robocomp/robocomp-teams.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div>
+    <div id="accordion-open" data-accordion="open">
+      <div v-for="(team, index) in teams" :key="index">
+        <h2 :id="`accordion-open-heading-${ index }`">
+          <button
+            type="button"
+            class="flex items-center justify-between w-full p-5 font-medium rtl:text-right text-gray-500 border border-b-0 border-gray-200 rounded-t-xl focus:ring-4 focus:ring-gray-200 dark:focus:ring-gray-800 dark:border-gray-700 dark:text-gray-400 hover:bg-gray-100 dark:hover:bg-gray-800 gap-3"
+            :data-accordion-target="`#accordion-open-body-${index}`"
+            aria-expanded="true"
+            :aria-controls="`accordion-open-body-${index}`"
+          >
+            <span>{{ team.name }}</span>
+            <svg
```

Comment:
for icons use [FontAwesome 6](https://fontawesome.com/search?o=r&m=free) instead of SVGs